app/uk/gov/hmrc/tariffclassificationfrontend/controllers/ReleaseCaseController.scala

```diff
@@ -41,14 +41,19 @@ class ReleaseCaseController @Inject()(authenticatedAction: AuthenticatedAction,
 
   private lazy val releaseCaseForm: Form[String] = ReleaseCaseForm.form
 
+  private def releaseCase(f: Form[String], caseRef: String)
```

Comment:
This code was repeated twice. 
I think now the code here is more readable and maintainable.